subprojects/gradle/build-verdict/src/test/kotlin/com/avito/android/build_verdict/BuildVerdictPluginTest.kt

```diff
@@ -12,13 +11,14 @@ import com.google.gson.GsonBuilder
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.io.TempDir
 import java.io.File
+import com.avito.android.build_verdict.RawBuildVerdictWriter.Companion.buildVerdictFileName as rawBuildVerdictFileName
 
 class BuildVerdictPluginTest {
```

Comment:
I decided to do not to cover the plain text verdict by tests.
Everything may change. I check readability by eyes ))